src/coordinated_workers/coordinator.py

```diff
@@ -785,6 +790,70 @@ def _reconcile_charm_labels(self) -> None:
             labels=self._coordinated_workers_solution_labels,
         )
 
+    def _get_cluster_scoped_mesh_policy(
+        self, source_model: str, source_application: str
+    ) -> List[MeshPolicy]:
+        """Return a mesh policy for a coordinator/worker to allow it to talk to all the other cluster units."""
```

Comment:
just a suggestion.  The "cluster_scoped" could mean a few things (for a while I thought we were talking about cluster scoped kubernetes resources)

```suggestion
    def _get_mesh_policy_into_cluster(
        self, source_model: str, source_application: str
    ) -> List[MeshPolicy]:
        """Return a mesh policy that allows a source to talk to all the other cluster units."""
```